### PR TITLE
Register fixes

### PR DESCRIPTION
**Pull request by ilzeLourens**

### CODE REVIEW

File: skietbaan_fe/src/components/Login.js [hunk 1]
```diff
@@ -157,6 +174,7 @@ class Login extends Component {
           }
         }
       }).catch(function (data) {
+        console.log("error")
```
Comment (FredJEhlers) [inline skietbaan_fe/src/components/Login.js]: remove debug logs

File: skietbaan_fe/src/components/Register.js [hunk 2]
```diff
@@ -186,6 +186,7 @@ class Register extends Component {
           window.location = "/home";
         }
       }).catch(function (data) {
+        console.log("error")
```
Comment (FredJEhlers) [inline skietbaan_fe/src/components/Register.js]: remove debug logs

File: skietbaan_fe/src/components/Login.js (hunk 1 above)
Comment (ilzeLourens) [reply skietbaan_fe/src/components/Login.js]: removed

File: skietbaan_fe/src/components/Register.js (hunk 2 above)
Comment (ilzeLourens) [reply skietbaan_fe/src/components/Register.js]: removed

**Pull request merged**